eng/build.ps1

```diff
@@ -263,7 +263,11 @@ foreach ($config in $configuration) {
   $argumentsWithConfig = $arguments + " -configuration $((Get-Culture).TextInfo.ToTitleCase($config))";
   foreach ($singleArch in $arch) {
     $argumentsWithArch =  "/p:TargetArchitecture=$singleArch " + $argumentsWithConfig
-    $env:__DistroRid="win-$singleArch"
+    if ($os -eq "Browser") {
+      $env:__DistroRid="browser-$singleArch"
+    } else {
```

Comment:
Do we want to also extend this for Android since it will need a similar fix if building Android on Windows (not WSL) will ever be supported?

Reply:
I think we can leave that till we support building android on windows. From my recent experience with adding support to build webassembly on windows, I guess it will need many other changes.